index.html

```diff
@@ -65,10 +71,10 @@ <h2 class="main-title">Tonic</h2>
                 <a class="see-project" href="#">See Project</a>
             </div>
         </div>
-        <div class="card">
-            <img class="card-img" src="./img/pic2.png" alt="first-project">
-            <h2 class="main-title">Multi-Post Stories</h2>
-            <div class="details-container">
+        <div class="card card-2">
+            <img class="card-img card-img-2" src="./img/pic2.png" alt="first-project">
+            <div class="details-container card-details-2">
+                <h2 class="main-title">Multi-Post Stories</h2>
```

Comment:
- [ ] Your cards should be wider in the desktop version as it is right now please try to use the right size as in Figma design in order to match it 
- Yours
![image](https://user-images.githubusercontent.com/61790587/177781323-f2fa5d3f-a547-42bc-89e9-38a7333a603c.png)

- Figma 
![image](https://user-images.githubusercontent.com/61790587/177781365-d5bb5bec-da83-49cd-9001-1066bd14a16a.png)
- 
![image](https://user-images.githubusercontent.com/61790587/177781433-ceaeab56-0bb4-4010-be3d-5bfafc3b05d2.png)